docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
       - containers-network
 
   mysql:
-    image: mysql:latest
+    image: mysql:8.0
```

Comment:
It's MySQL 8.0.36 on the server. 
Be careful with Docker version uplifting `cause that could bring inconsistently to the build with the server's configuration.